src/projects/projects.controller.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpException,
+  HttpStatus,
+  Param,
+  Post,
+  Put,
+  Req,
+  UseGuards,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { UserGuard } from '../user/user.guard';
+import { ProjectsService } from './projects.service';
+import { UserJwtPayload } from '../user/user.service';
+import { CreateProjectDto } from './dtos/CreateProject.dto';
+import { UpdateProjectDto } from './dtos/UpdateProject.dto';
+
+@Controller('api/project')
+export class ProjectsController {
+  constructor(private readonly projectsService: ProjectsService) {}
+
+  @UseGuards(UserGuard)
+  @Get()
+  @HttpCode(200)
+  async loadProjects(@Req() req: Request & { user: UserJwtPayload }) {
+    const { id } = req.user;
+    const projects = await this.projectsService.getProjects(id);
+    return {
+      statusCode: 200,
+      message: 'Projects loaded successfully.',
+      projects: projects,
+    };
+  }
+
+  @UseGuards(UserGuard)
+  @Get(':id')
+  @HttpCode(200)
+  async loadProject(
+    @Param('id') projectId: number,
+    @Req() req: Request & { user: UserJwtPayload },
+  ) {
+    const { id: userId } = req.user;
+    const project = await this.projectsService.getProject(userId, projectId);
+    return {
+      statusCode: 200,
+      message: 'Project loaded successfully.',
+      project: project,
+    };
+  }
+
+  @UseGuards(UserGuard)
+  @Post()
+  @HttpCode(201)
```

Comment:
(praise): Nice attention to detail, using a `201` status code for a `POST` request.

(suggestion; non-blocking): This is another great opportunity to use the `HttpStatus` enum to replace these magic numbers with something more self-descriptive.